code/game/machinery/bots/mulebot.dm

```diff
@@ -390,23 +391,17 @@ var/global/mulebot_count = 0
 	else
 		to_chat(user, "<span class='warning'>Access denied.</span>")
 		return 0
-
-// mousedrop a crate to load the bot
 // can load anything if emagged
 
-/obj/machinery/bot/mulebot/MouseDrop_T(var/atom/movable/C, mob/user)
 
-	if(user.stat)
-		return
+// Mousedrop a crate to load the bot
 
-	if (!on || !istype(C)|| C.anchored || get_dist(user, src) > 1 || get_dist(src,C) > 1 )
-		return
-
-	if(load)
-		return
+/obj/machinery/bot/mulebot/MouseDrop_T(atom/dropping, mob/user)
+	load(dropping)
```

Comment:
Both this and the other mousedrop should have sanity just for the state of the user, ie if(user.stat) return